src/codewhisperer/util/supplementalContext/utgUtils.ts

```diff
@@ -51,6 +53,10 @@ export async function fetchSupplementalContextForTest(
         return undefined
     }
 
+    if (CodeWhispererUserGroupSettings.instance.userGroup !== UserGroup.Utg) {
+        return []
+    }
```

Comment:
Not returning `undefined` on purpose, we need supplemental context relevant fields sent no matter what userGroup the user belongs.